### PR TITLE
MapFragment Javadocs

### PR DESCRIPTION
**Pull request by WJXHenry**
Part of #39 

**Summary**

This PR includes the Javadocs for the `MapFragment` implementation so far.

### CODE REVIEW

File: Moodlet/app/src/main/java/com/cmput3owo1/moodlet/fragments/MapFragment.java [hunk 1]
```diff
@@ -23,64 +23,97 @@
     private static final String TAG = "Moodlet";
```
Comment (bojotamara) [inline Moodlet/app/src/main/java/com/cmput3owo1/moodlet/fragments/MapFragment.java:27]: I think each file needs an overarching comment as well

Comment (bojotamara) [reply Moodlet/app/src/main/java/com/cmput3owo1/moodlet/fragments/MapFragment.java:27]: Each class i mean

Comment (WJXHenry) [reply Moodlet/app/src/main/java/com/cmput3owo1/moodlet/fragments/MapFragment.java:27]: Added description to the MapFragment class

File: Moodlet/app/src/main/java/com/cmput3owo1/moodlet/fragments/MapFragment.java [hunk 2]
```diff
@@ -18,6 +18,10 @@
 import com.google.android.gms.maps.model.LatLng;
 import com.google.android.gms.maps.model.MapStyleOptions;
 
+/**
+ * A fragment that holds a Google API MapView that displays the location of the user's mood event
+ * history and optionally displays the location of the user's followers' mose recent mood event.
```
Comment (bojotamara) [inline Moodlet/app/src/main/java/com/cmput3owo1/moodlet/fragments/MapFragment.java]: Typo - "mose recent"

Comment (WJXHenry) [reply Moodlet/app/src/main/java/com/cmput3owo1/moodlet/fragments/MapFragment.java]: Nice

Comment (WJXHenry) [reply Moodlet/app/src/main/java/com/cmput3owo1/moodlet/fragments/MapFragment.java]: Fixed typo lol

**Review by bojotamara — APPROVED** [review]
merge away

**Pull request merged**